main.js

```diff
@@ -38,141 +38,31 @@ menuLinks.forEach((link) => {
 // &
 // PopUp Window
 
-const cardsContent = {
-  images: {
-    m_d_popup: [
-      [
-        // Content in some array follows this order
-        // [mobile, mobilePopup, desktop, desktopPopup]
-        //               ('') empty values mean there no content for that Part
-        'images/cards/card1.svg', 'images/cards/card_1_popup.svg',
-        'images/cards/card1.svg', 'images/cards/card_1_popup.svg',
-      ],
-      [
-        '', 'images/cards/card_2&5_desktop_light.svg',
-        '', 'images/cards/card_2&5_desktop_light.svg',
-      ],
-      [
-        '', 'images/cards/card_3&6_desktop_light.svg',
-        '', 'images/cards/card_3&6_desktop_light.svg',
-      ],
-      [
-        '', 'images/cards/card_4&7_desktop_light.svg',
-        '', 'images/cards/card_4&7_desktop_light.svg',
-      ],
-      [
-        '', 'images/cards/card_2&5_desktop_light.svg',
-        '', 'images/cards/card_2&5_desktop_light.svg',
-      ],
-      [
-        '', 'images/cards/card_3&6_desktop_light.svg',
-        '', 'images/cards/card_3&6_desktop_light.svg',
-      ],
-      [
-        '', 'images/cards/card_4&7_desktop_light.svg',
-        '', 'images/cards/card_4&7_desktop_light.svg',
-      ],
+const cardsContent = [
+  // Content in some array follows this order
+  // [mobile, mobilePopup, desktop, desktopPopup]
+  // ('') empty values mean there no content for that Part
+  cardsContent[ind][header][mobile]
+  {
+    header: [
+      'Multi-Post Stories', 'Multi Post Stories',
+      'Multi-Post Stories', 'Keeping track of hundreds  of components website',
```

Comment:
- [ ] This `cardsContent` should be an array of objects which store objects containing the information for each project. Each of the objects should contain at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code. What you've done here is grouping all the `headers` together, `descriptions` together and so on which should not be the case.
- Your array should look like this:
![Annotation 2022-03-05 065014](https://user-images.githubusercontent.com/85113567/156870200-985544db-95c5-421e-a4eb-7b62afc9fe5a.jpg)
- Write all the information about one project inside a single object before opening another object and writing the information about the next project 👍

Reply:
OMG This is not possible, I haven't grouped headers and descriptions together, if you didn't realize in the comment //[mobile, mobilePopup, desktop, desktopPopup]
says that arrays that have four values, each one belongs to different content, the origin design changes some headers paragraphs and images when in mobile, desktop and when mobilePopup, desktopPopup

Reply:
Hello @Jord4N-AC are you available for a zoom call so that you can better explain yourself?